pygenometracks/tracks/BedGraphTrack.py

```diff
@@ -184,18 +184,28 @@ def plot(self, ax, chrom_region, start_region, end_region):
         score_list, pos_list = self.get_scores(chrom_region, start_region, end_region)
         score_list = [float(x[0]) for x in score_list]
 
-        # the following two lines will convert the score_list and the
-        # tuples in pos list (where is item is tuple(start, end)
-        # into an x value (pos_list) and a y value (score_list)
-        # where x = start1, end1, star2, end2 ...
-        # and y = score1, score1, score2, score2 ...
-
-        # convert [1, 2, 3 ...] in [1, 1, 2, 2, 3, 3 ...]
-        score_list = np.repeat(score_list, 2)
-        if self.properties['nans to zeros']:
-            score_list[np.isnan(score_list)] = 0
-        # convert [(0, 10), (10, 20), (20, 30)] into [0, 10, 10, 20, 20, 30]
-        x_values = np.asarray(sum(pos_list, tuple()), dtype=np.float)
+        if 'use middle' in self.properties and self.properties['use middle'] == 'yes':
```

Comment:
You could rewrite this to `self.properties.get('use middle', False) == 'yes'`

Reply:
Yes, I did not think about it. I will integrate.